### DISCUSSION TITLE
Private profile

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19159] [Other Feature Feedback, Questions, & Ideas] [answered]
I just wanted to offer some positive feedback on the private profile option: https://github.blog/changelog/2022-04-21-private-profiles/. I think it's great that this has been added, does exactly what I want! And I think with me a lot of others enjoy the freedom of disabling the public profile, just like gitlab users have been able to do for a while now: https://docs.gitlab.com/ee/user/profile/#make-your-user-profile-page-private.

See also these related issues and discussions:

- https://github.com/isaacs/github/issues/142
- https://github.com/isaacs/github/issues/627
- https://github.com/orgs/github-community/discussions/4098

Though not exactly similar, I feel they do illustrate that some people like to have the option of hiding (some of) their profile info. I appreciate that we now have control over this as users. I realize this is not exactly a discussion, but this is where you're encouraged to leave feedback, so I hope this kind of positive feedback is appreciated as well.

**Comment by rodukov** [comment, marked as answer]
I agree with you. **Everyone is free to choose which profile to have private or public**. The company must provide a choice. For example, I prefer to have open profile. If the answer helped, you can make it the main one.

**Reply by ghost** [reply]
> If the answer helped, you can make it the main one

👍️ sounds good!

**Reply by vojkny** [reply]
also discussed here: https://github.com/orgs/community/discussions/31372

I don't think the "answerred" makes any difference. I think these discussions are rather left ignored :/

**Comment by ghost** [comment]
LGTM!

**Comment by Maher-Reven** [comment]
LGTM!
